package.json

```
@@ -45,5 +45,7 @@
     "react-textarea-autosize": "^7.1.2",
     "uuid-v4": "^0.1.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "bluetooth-uuid-database": "github:NordicSemiconductor/bluetooth-numbers-database"
```

With this dependency, always the latest version of [`bluetooth-numbers-database`](https://github.com/NordicSemiconductor/bluetooth-numbers-database) is used. 

This could lead to several troubles, one being that building `pc-nrfconnect-ble` today and tomorrow would yield different results if `bluetooth-numbers-database` is updated in between. That would make tracking down problems harder, because it would be hard to understand how a certain version of this app was built. If one version of `bluetooth-numbers-database` contains errors this could also mean that `pc-nrfconnect-ble` is broken, while nothing was changed here.

Because of all of this, I suggest that we target a specific commit of `bluetooth-numbers-database`, e.g. like this:
```suggestion
    "bluetooth-uuid-database": "github:NordicSemiconductor/bluetooth-numbers-database#bea7edc8aa"
```

@bencefr: What do you think?

---

I agree, so committed the suggestion, however we are still updating the module and we'll need to finalise the dependency here later.